docs/code-quality/ca2241.md

```diff
@@ -68,6 +68,13 @@ Examples:
 
 You can configure this option for just this rule, for all rules, or for all rules in this category (Usage). For more information, see [Configure FxCop analyzers](configure-fxcop-analyzers.md).
 
+Instead of a specifying a long list of additional string formatting methods, you can configure the analyzer to try and determine this automatically. By default, this option is disabled.
```

Comment:
```suggestion
Alternatively, instead of specifying an explicit list of additional string formatting methods, you can configure the analyzer to try and determine this automatically. By default, this option is disabled.
```